redfish-exporter/listener.go

```diff
@@ -185,10 +187,16 @@ func processRequest(AppConfig Config, conn net.Conn, req *http.Request, eventCou
 		log.Printf("Message Args: %v", messageArgs)
 		log.Printf("Origin Of Condition: %s", originOfCondition)
 		for _, triggerEvent := range AppConfig.TriggerEvents {
-			if eventType == triggerEvent.EventType && eventId == triggerEvent.EventId && severity == triggerEvent.Severity {
```

Comment:
are we not going to check event severity?

Reply:
Yeah, the type and severity is pre-defined as part of event ID. So, we do not need to explicitly check it.